### DISCUSSION TITLE
[React Native] react native에서 apk 파일 다운 받기 (in MAC)

### DISCUSSION THREAD

**Discussion by publdaze** [BDD-CLUB/challenge-every-week-posting#66] [1. 개발 관련 글 모음]
글 올리려고 왔더니 바로 앞에 서윤님 같은 내용으로 먼저 올려주셨었네요...ㅎㅎ
저는 딱히 상세 설명은 없고 순서만 띡띡 적어놓은 글입니다...!
나중에 설명도 한 번 추가해볼게요!
서윤님과 같은 내용이긴 하지만 방법은 다르답니다~! 😄

[노션 링크](https://sparkling-radium-6f4.notion.site/react-native-apk-in-MAC-faae0945fc11466eadddd452cc7ee914?pvs=4)

**Comment by pipisebastian** [comment]
ㅋㅎㅋㅎㅋㅎㅋ오홍!! 바로 apk로 하는 방법이..! 두둥..! 나중에 따라해보겠숩니다~~
